### DISCUSSION TITLE
Compatibility for 1.21

### DISCUSSION THREAD

**Discussion by Yessir373** [iPortalTeam/ImmersivePortalsMod#1621] [Compatibility Issue Report]
### If it's a mod compatibility issue, please provide the link(s) to the incompatible mod(s)

https://modrinth.com/mod/immersiveportals

### Minecraft version

Other (specify below)

### Describe the issue

I don't know if this is a compatibility issue or not but i want to ask to add the version 1.21 on CurseForge and Modrinth and not just version 1.21.1

**Comment by ImSpaceLover** [comment]
1.21.1 version should work with 1.21

**Reply by Yessir373** [reply]
Okay but i have a problem with it, on the version 6.0.2 without compatibility render mode the entity bugs out and players when looking at portals, and on 6.03 the world render bugs out without it but when i turn on the compatibility render mode the portal looks ugly and the entities in the other side of portals the mobs animation just lags out
